packages/render-fidelity-tools/test/config.json

```diff
@@ -766,8 +766,7 @@
       },
       "exclude": [
         "gltfpt",
-        "babylon",
-        "filament"
+        "babylon"
```

Comment:
after updating filament, there's no errors on this scenario anymore.

Reply:
Thanks!